src/fsharp/FSharp.Compiler/FSharp.Compiler.fsproj

```diff
@@ -532,7 +532,7 @@
     <Reference Include="System.Numerics" />
     <Reference Include="ISymWrapper, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
     <Reference Include="Microsoft.DiaSymReader.PortablePdb">
-      <HintPath>..\..\..\packages\Microsoft.DiaSymReader.PortablePdb.1.2.0\lib\portable-net45+win8\Microsoft.DiaSymReader.PortablePdb.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.DiaSymReader.PortablePdb.1.1.0\lib\portable-net45+win8\Microsoft.DiaSymReader.PortablePdb.dll</HintPath>
```

Comment:
@KevinRansom Please check this, I see this in packages.config:
```
  <package id="Microsoft.DiaSymReader.PortablePdb" version="1.1.0"  />
```
This was causing a build failure in fsharp/fsharp